.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ "mongo-workflow-fix" ]
```

Comment:
OK for testing, but should get changed to master just before merge

Reply:
Fixed with [0dc0853](https://github.com/SciCatProject/oai-provider-service/pull/112/commits/0dc085392036af780768ba19704ca479c0e3dfaf)